<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>post</class>
 <widget class="QDialog" name="post">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 25, 40);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Unispace&quot;;
Border-radius: 10px;
background-color: rgb(85, 0, 255);
text-align:centre;
padding: 5px</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>581</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:centre;
Border-radius: 10px;
background-color: rgb(141, 1, 255);
font: 700 12pt &quot;Unispace&quot;;
padding: 5px;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>801</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Unispace&quot;;
Border-radius: 10px;
background-color: rgb(85, 0, 255);
text-align:centre;
padding: 5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="like">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>280</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
font: 700 16pt &quot;Unispace&quot;;
border-radius:10px;
}

QPushButton:Hover{
	background-color: rgb(170, 0, 255);
}
</string>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QPushButton" name="postComment">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>520</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{font: 700 19pt &quot;Unispace&quot;;
Border-radius: 10px;
background-color: rgb(255, 0, 255);}
QPushButton:Hover{
	
	background-color: rgb(255, 0, 127);
}
</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:Hover{
	background-color: rgb(85, 0, 255);
}

QPushButton{background-color: rgb(255, 0, 127);
font: 700 22pt &quot;Unispace&quot;;
Border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:centre;
Border-radius: 10px;
background-color: rgb(141, 1, 255);
font: 700 12pt &quot;Unispace&quot;;
padding: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 127);
font: 700 16pt &quot;Unispace&quot;;</string>
   </property>
   <property name="text">
    <string>Drop a Comment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>80</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Unispace&quot;;
Border-radius: 10px;
background-color: rgb(85, 0, 255);
text-align:centre;
padding: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>40</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
font: 700 16pt &quot;Unispace&quot;;</string>
   </property>
   <property name="text">
    <string>Liked By</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
font: 700 18pt &quot;Unispace&quot;;
</string>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="noOfCom">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Unispace&quot;;
Border-radius: 10px;
text-align:centre;
background-color: rgb(255, 0, 127);
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Unispace&quot;;
Border-radius: 10px;
background-color: rgb(85, 0, 255);
text-align:centre;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/yapp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
font: 700 16pt &quot;Unispace&quot;;</string>
   </property>
   <property name="text">
    <string>Activity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>110</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 127);
font: 700 16pt &quot;Unispace&quot;;</string>
   </property>
   <property name="text">
    <string>Content</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
font: 700 16pt &quot;Unispace&quot;;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
